### PR TITLE
Add schedule

### PR DESCRIPTION
**Pull request by joehuchette**
I'll leave the Spanish translation to a more qualified person :)

### CODE REVIEW

File: meetings/santiago2019/index.md [hunk 1]
```diff
@@ -25,39 +24,66 @@ The workshop will include hands-on tutorials (ranging from no-experience
...
-for a flavor of the workshop.
+On Tuesday morning we will hold an introductory tutorial on Julia and JuMP; no prior experience necessary! Additionally, on Wednesday afternoon we will hold an advanced tutorial (tentative topics include MOI, JuMP extensions, and solver interfaces). The tutorials will be open to anyone, even if they have not attended the workshop.
 
-**To register your attendance at the workshop, (and optionally submit a proposal
-for a talk), [use this form](https://goo.gl/forms/2nEljULHwC3zzcE13).**
+## Tentative Schedule
 
-Talk proposals are welcome until ~~December 21st~~ **January 4th** and talk acceptance will be 
-notified by ~~January 4th~~ **January 11th**. 
+_Note: This schedule is subject to change!_
```
Comment (mlubin) [inline meetings/santiago2019/index.md]: (Check here for updates.)

File: meetings/santiago2019/index.md [hunk 2]
```diff
@@ -25,39 +24,66 @@ The workshop will include hands-on tutorials (ranging from no-experience
...
 
-## Location
+### Tuesday
+
+<tt>10:00 - 11:30</tt> **Introductory Tutorial**
+
+<tt>13:00 - 14:30</tt> **Keynotes**
+* Stefan Karpinski (Julia Computing)
+* _Milestones and plans for JuMP 1.0_, Miles Lubin (Google)
```
Comment (mlubin) [inline meetings/santiago2019/index.md]: Could we change the title of my talk to "The roadmap for JuMP 1.0"?

File: meetings/santiago2019/index.md [hunk 3]
```diff
@@ -25,39 +24,66 @@ The workshop will include hands-on tutorials (ranging from no-experience
...
+* _A Julia/JuMP based Integrated Energy Resource Planning Model_, Alessandro Soares (PSR)
+* _PowerSimulations.jl a Integrated and Scalable Power Systems modeling tool_, Jose Daniel Lara (UC Berkeley / NREL)
+
+<tt>16:00 - EOD</tt> **Collaboration time**
+
+### Wednesday
+
+<tt>9:00 - 10:25</tt> **Solvers**
+* _COSMO - A conic operator splitting method for large conic problems_, Michael Garstka (University of Oxford)
+* _the Hypatia.jl solver: conic interior point algorithms and interfaces_, Chris Coey (MIT)
```
Comment (mlubin) [inline meetings/santiago2019/index.md]: "the" -> "The"

File: meetings/santiago2019/index.md [hunk 4]
```diff
@@ -25,39 +24,66 @@ The workshop will include hands-on tutorials (ranging from no-experience
...
+<tt>10:50 - 12:10</tt> **Applications**
+* _Power Systems Modeling using JuMP and Julia_, Alvaro González (Skolkovo Institute of Science and Technology)
+* _POD.jl for Provably Optimal Quantum Circuit Design_, Harsha Nagarajan (Los Alamos National Laboratory)
+* _Applications of SOS in Flowpipe Construction_, Marcelo Forets (CURE)
+* _Solving Multi-period Mine Planning Models with Endogenous Uncertainty: Using Julia with JuMP_, Tomas Lagos Gonzalez (Universidad de Chile)
+* _SDDPHYDRO.jl - A Hydrothermal Dispatch Julia Package_, Andrew David Werner Rosemberg (PUC-Rio)
+
+<tt>14:00 - 15:30</tt> **Advanced tutorial**
+
+<tt>15:30 - EOD</tt> **Collaboration time**
```
Comment (mlubin) [inline meetings/santiago2019/index.md]: Collaboration time and core developer meeting?

File: meetings/santiago2019/index.md [hunk 5]
```diff
@@ -25,39 +24,66 @@ The workshop will include hands-on tutorials (ranging from no-experience
...
+
+<tt>10:50 - 12:00</tt> **Modeling paradigms**
+* _Graph-Based Modeling and Optimization using Plasmo.jl_, Jordan Jalving (University of Wisconsin-Madison)
+* _Set Programming with JuMP_, Benoît Legat (UC Louvain)
+* _ParameterJuMP.jl_, Joaquim Dias Garcia (PSR / PUC-Rio)
+* _JuliaMoments_, Tillmann Weisser (Los Alamos National Laboratory)
 
-The workshop will be held in Santiago, Chile. The exact venue details will be
-released after talk acceptance.
+<tt>14:00 - EOD</tt> Collaboration time
```
Comment (blegat) [inline meetings/santiago2019/index.md]: Collaboration time should be in bold

**Comment by joehuchette** [comment]
Could a Spanish speaker provide a translation?

**Comment by mlubin** [comment]
Note: we may not want to duplicate the schedule as it would be easy to get out of sync. IMO it's better for the spanish version to refer to the schedule in the english part.

**Pull request merged**